.github/workflows/dev-long-tests.yml

```diff
@@ -150,6 +150,88 @@ jobs:
         make clean
         make -C tests test-fuzzer-stackmode
 
+  mingw-long-test:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          { compiler: gcc, platform: x64, action: build, script: "make allzstd MOREFLAGS=-static", artifact: true},
+          { compiler: gcc, platform: x86, action: build, script: "make allzstd MOREFLAGS=-static", artifact: true},
+          { compiler: clang, platform: x64, action: build, script: "MOREFLAGS='--target=x86_64-w64-mingw32 -Werror -Wconversion -Wno-sign-conversion -Wno-unused-command-line-argument -Wno-implicit-int-float-conversion' make -j allzstd V=1", artifact: false},
+          { compiler: gcc, platform: x64, action: test, script: "", artifact: false},
+        ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Mingw long test
+      run: |
+        $env:PATH_ORIGINAL = $env:PATH
+        $env:PATH_MINGW32 = "C:\msys64\mingw32\bin"
+        $env:PATH_MINGW64 = "C:\msys64\mingw64\bin"
+        COPY C:\msys64\usr\bin\make.exe C:\msys64\mingw32\bin\make.exe
+        COPY C:\msys64\usr\bin\make.exe C:\msys64\mingw64\bin\make.exe
+        IF ("${{matrix.platform}}" -eq "x64")
+        {
+            $env:PATH = $env:PATH_MINGW64 + ";" + $env:PATH_ORIGINAL
+        }
+        ELSEIF ("${{matrix.platform}}" -eq "x86")
+        {
+            $env:PATH = $env:PATH_MINGW32 + ";" + $env:PATH_ORIGINAL
+        }
+        IF ("${{matrix.action}}" -eq "build")
+        {
+          make -v
+          sh -c "${{matrix.compiler}} -v"
+          ECHO "Building zlib to static link"
+          $env:CC = "${{matrix.compiler}}"
+          sh -c "cd .. && git clone --depth 1 --branch v1.2.11 https://github.com/madler/zlib"
+          sh -c "cd ../zlib && make -f win32/Makefile.gcc libz.a"
+          ECHO "Building zstd"
+          $env:CPPFLAGS = "-I../../zlib"
+          $env:LDFLAGS = "../../zlib/libz.a"
+          sh -c "${{matrix.script}}"
+          IF ("${{matrix.artifact}}" -eq "true")
+          {
+            ECHO Creating artifacts
```

Comment:
OK, so that's something I overlooked regarding this topic.

On top of Windows tests, Appveyor was also used to create the Windows binary packages at release time.
It's my understanding that you transferred this responsibility to Github Actions.

How does it work ?
In which cases are artifact created, and how to access them ?

For example, I note that, in this PR, there is a script section called "upload artifacts", but it's empty (runtime 0s).

Reply:
I think a direct translation from the AppVeyor release artifact process doesn't work, since it seems like AppVeyor will just display any artifacts in the `\artifacts` folder, whereas I _think_ GH Actions works a bit differently.

There's some documentation here: https://github.com/actions/upload-artifact on how to upload the artifact, it seems you just need an extra step that points to the artifact/directory you'd like to upload.

Reply:
Ah I thought I'd gotten it working, I've reverted those tests back to appveyor for now and will try to get them working on GH in a followup.

Reply:
The code looks correct to me, 
but just to be on the safe side, have you also been able to test that the artifacts are still generated on Appveyor ?

Reply:
Artifacts on appveyor branch PR look good:

https://ci.appveyor.com/project/YannCollet/zstd-p0yf0/builds/41368517/job/kckbm0scodyuxp4e/artifacts
https://ci.appveyor.com/project/YannCollet/zstd-p0yf0/builds/41368517/job/s1c9o37ux7qkn38s/artifacts

Reply:
Excellent ! Thanks @binhdvo !

Reply:
Hm, there are some appveyor failures now, but they don't look related to these changes, investigating.